endpoints/setuid.go

```diff
@@ -185,6 +199,10 @@ func NewSetUIDEndpoint(cfg *config.Configuration, syncersByBidder map[string]use
 			w.Header().Add("Content-Length", "0")
 			w.WriteHeader(http.StatusOK)
 		}
+		// Add Warning to Response If Present
+		if err != nil && err.Error() == errSyncerIsNotPriority.Error() {
+			w.Write([]byte("Warning: " + err.Error() + " returning unaltered cookie"))
```

Comment:
Do we need to return the cookie if it is not being altered? Not returning any cookie at all should result in the original cookie remaining in place.

Reply:
Great catch, I just pushed an update where we simply return with a 200 and a warning attached to the response when we encounter this error scenario.